source/modulo_components/include/modulo_components/ComponentInterface.hpp

```diff
@@ -738,12 +738,10 @@ template<class NodeT>
 inline std::string ComponentInterface<NodeT>::validate_input_signal_name(const std::string& signal_name) {
   std::string parsed_signal_name = utilities::parse_topic_name(signal_name);
   if (parsed_signal_name.empty()) {
-    throw exceptions::AddSignalException(
-        "Failed to add input '" + signal_name + "': Parsed signal name is empty."
-    );
+    throw exceptions::AddSignalException("Parsed signal name is empty.");
   }
   if (this->inputs_.find(parsed_signal_name) != this->inputs_.cend()) {
-    throw exceptions::AddSignalException("Failed to add input '" + signal_name + "': Input already exists");
+    throw exceptions::AddSignalException("Input already exists");
```

Comment:
You don't think it's worth keeping the `signal_name` in the error message?

Reply:
I dont have it in front of me but if I remember correctly from yesterday, this exception is caught a few lines further down and the log message contains the signal name. So if I dont remove that, you end up with messages like "Failed to add input 'test': AddSignalException: Failed to add input 'test'"

Reply:
Thanks for the clarification, that was easy to miss just looking at this diff - in that case I definitely appreciate the reduction in code duplication!